assignment1/q1.cpp

```diff
@@ -0,0 +1,20 @@
+bool permutation(string base, string test) {
+
+    if(base.size() != test.size())
+        return false; // not permutations for sure
+
+    int letters[256]; //array for ascii chars
```

Comment:
When programming in C++, I prefer using a vector rather than a C-style array.
You can also set a default value to a vector:
`vector<int> letters(256, 0)`
`256` is the size, `0` is the default value.